doctor/views.py

```
@@ -0,0 +1,339 @@
+                status=status.HTTP_201_CREATED
+            )
+        else:
+            return Response(
+                api_ingress.build_doctor_error_response(
+                    'validation',
+                    details=result
+                ),
+                status=status.HTTP_400_BAD_REQUEST
+            )
+            
+    except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)